Coffee_Time.sql

```diff
@@ -78,7 +78,8 @@ CREATE TABLE Products(
 CREATE TABLE Payments(
 	id_payment INT PRIMARY KEY,
 	payment_type NVARCHAR(4) CHECK(payment_type='CASH' OR payment_type='CARD') NOT NULL,
-    id_schedule INT FOREIGN KEY REFERENCES Employers_Schedule(id_Schedule)
+	payment_date DATETIME NOT NULL,
+    id_subsidiary INT FOREIGN KEY REFERENCES Subsidiaries(id_Subsidiary)

```

Comment:
Add one more tab

Reply:
What do you mean ?

Reply:
Should be at the same level
![image](https://user-images.githubusercontent.com/52528166/151672990-5cc6aacb-77ef-4e78-9ce6-80fccd0242d3.png)

Reply:
They are at same level
![image](https://user-images.githubusercontent.com/58091611/151673034-2f3effbf-21cd-4459-a39e-609f12a9e2fb.png)